examples/GrpcCli/Sources/GrpcCli/GrpcCli.swift

```diff
@@ -0,0 +1,153 @@
+import NIOPosix
+
+struct GrpcOptions: ParsableArguments {
+    @Option(help: "IP or DNS name of the Node.")
+    var host = "localhost"
+
+    @Option(help: "Port of the Node.")
+    var port = 20000
+
+    var target: ConnectionTarget {
+        ConnectionTarget.host(host, port: port)
+    }
+}
+
+struct BlockOption: ParsableArguments {
+    @Option(help: "Hash of the block to query against. Defaults to last finalized block.")
+    var blockHash: String?
+
+    var block: BlockIdentifier {
+        get throws {
+            if let blockHash {
+                return try .hash(BlockHash(fromHexString: blockHash))
+            }
+            return .lastFinal
+        }
+    }
+}
+
+struct AccountOption: ParsableArguments {
+    @Argument(help: "Address of the account to inspect.")
+    var accountAddress: String
+
+    var address: AccountAddress {
+        get throws {
+            try AccountAddress(base58Check: accountAddress)
+        }
+    }
+
+    var identifier: AccountIdentifier {
+        get throws {
+            try .address(address)
+        }
+    }
+}
+
+@main
+struct GrpcCli: AsyncParsableCommand {
+    @OptionGroup
+    var options: GrpcOptions
+
+    static var configuration = CommandConfiguration(
+        abstract: "A CLI for demonstrating and testing use of the gRPC client of the SDK.",
+        version: "1.0.0",
+        subcommands: [CryptographicParameters.self, Account.self]
+    )
+
+    struct CryptographicParameters: AsyncParsableCommand {
+        static var configuration = CommandConfiguration(
+            abstract: "Display the cryptographic parameters of the chain."
+        )
+
+        @OptionGroup
+        var root: GrpcCli
+
+        @OptionGroup
+        var block: BlockOption
+
+        func run() async throws {
+            let res = try await withClient(target: root.options.target) {
+                try await $0.getCryptographicParameters(
+                    at: block.block
+                )
+            }
+            print(res)
+        }
+    }
+
+    struct Account: AsyncParsableCommand {
+        static var configuration = CommandConfiguration(
+            abstract: "Subcommands related to a particular account.",
+            subcommands: [NextSequenceNumber.self, Info.self]
+        )
+
+        @OptionGroup
+        var account: AccountOption
+
+        struct NextSequenceNumber: AsyncParsableCommand {
+            static var configuration = CommandConfiguration(
+                abstract: "Display the next sequence number of the provided account."
+            )
+
+            @OptionGroup
+            var root: GrpcCli
+
+            @OptionGroup
+            var account: Account
+
+            func run() async throws {
+                let res = try await withClient(target: root.options.target) {
+                    try await $0.getNextAccountSequenceNumber(
+                        of: account.account.address
+                    )
+                }
+                print(res)
```

Comment:
Maybe add a bit of context (e.g., as a prefix) to the print statement

Reply:
Well it's the result of the command that the user issued. What prefix do you have in mind?

Reply:
If you have a script that runs several commands, then it might be helpful to have something that delimits the results of the individual commands, like a capital letter method name as the prefix and and a newline as a postfix

Reply:
I would say it's up to the script to ensure that its output is legible. Making sure that the tools it use output as little as possible is a great help in that regard.

Reply:
I should add that as `test.sh` has tracing output enabled (via the `x` in `set -eux`) it will print the commands that it execute before the output, ensuring that all the relevant information is there (even if it isn't super pretty).